components/brave_shields/core/common/brave_shields.mojom

```diff
@@ -14,6 +37,11 @@ interface BraveShieldsHost {
   // Notify the browser process that JavaScript execution has been temporary
   // allowed, passing the details in |details| as a 16-bit string.
   OnJavaScriptAllowedOnce(mojo_base.mojom.String16 details);
+  // As the browser process whether a particular website should be has a
+  // webcompat exception
+  [Sync]
```

Comment:
Sync mojom call is *really* not a good thing. I haven't look at the renderer code yet, but probably we will need to proactively push exceptions to renderer

Reply:
I have moved this to a content settings provider instead.